Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -43,6 +44,78 @@ YUI.add('ez-locationviewviewservice', function (Y) {
         },
 
         /**
+         * moveAction event handler, launch the universal discovery widget
+         * to choose a location to move the content
+         *
+         * @method _selectLocation
+         * @protected
+         * @param {EventFacade} e
+         */
+        _selectLocation: function (e) {
+            this.fire('contentDiscover', {
+                config: {
+                    title: "Select the location you want to move your content into",
+                    contentDiscoveredHandler: Y.bind(this._moveContent, this),
+                },
+            });
+        },
+
+        /**
+         * Move the content to the selected location
+         *
+         * @method _moveContent
+         * @protected
+         * @param {EventFacade} e
+         */
+        _moveContent: function (e) {
+            var app = this.get('app'),
+                initialActiveView = app.get('activeView'),
+                parentLocationId = e.selection.location.get('id'),
+                locationId = this.get('location').get('id'),
+                that = this,
+                notificationIdentifier =  'move-notification-' + parentLocationId + locationId;
+
+            this._fireNotify('The move operation of your content has begun', notificationIdentifier, 'started', 5);
+
+            this.get('location').move({api: this.get('capi')}, parentLocationId, function (error, response) {
+                if (error) {
+                    that._fireNotify('An error occured while moving your content', notificationIdentifier, 'error', 5);
+                    return;
+                }
+
+                that._fireNotify('Your content has been successfully moved', notificationIdentifier, 'done', 5);
+
+                if ( app.get('activeView') === initialActiveView ) {
+                    app.navigateTo(
+                        'viewLocation',
+                        {id: response.getHeader('location')}
+                    );
+                }
+            });
+        },
+
+        /**
+         * Fire 'notify' event
+         *
+         * @method _fireNotify
+         * @protected
+         * @param {String} text the text shown durring the notification
+         * @param {String} identifier the identifier of the notification
+         * @param {String} state the state of the notification
+         * @param {Integer} timeout the number of second, the notification will be shown
+         */
+        _fireNotify: function (text, identifier, state, timeout) {
```

Comment:
didn't you change the name yesterday ? `_fireNotification` or `_notify` would be a better name